package.json

```diff
@@ -23,11 +23,12 @@
   ],
   "scripts": {
     "bootstrap": "yarn run build:constants && yarn --cwd example && yarn && yarn pods",
-    "build:constants": "node ./scripts/constants.js",
-    "build:declaration": "tsc --project tsconfig.declaration.json",
     "build:api:extractor": "api-extractor run --local",
     "build:api:documenter": "api-documenter markdown --input-folder ./temp --output-folder ./docs",
+    "build:constants": "node ./scripts/constants.js",
+    "build:declaration": "tsc --project tsconfig.declaration.json",
     "build:docs": "yarn run prepare && yarn run build:api:extractor && yarn run build:api:documenter",
+    "build:errors": "node ./scripts/errors.js",
```

Comment:
Because `yarn run build:errors` has a dependency on the `@twliio/voice-errors` module, we can't have this step run during `boostrap` since that runs before `yarn` installs anything. We're also committing the generated-errors file, does that affect our desire to put it in the prepare step? I think it's appropriate to have it part of the `check` script since that is run on git-pre-commit and will help check things just in case.

Reply:
Got it. You're right we already commit generated errors so no need to run this automatically.